setup.cfg

```diff
@@ -50,17 +47,15 @@ ignore =
     W503,
     # Newline after binary operator. Ignored by default (which we want to keep)
     W504,
+    # No need for coding=utf-8 anymore
+    C101
```

Comment:
Instead of this, change `accept-encodings: utf-8` below to [`no-accept-encodings: True`](https://github.com/tk0miya/flake8-coding#no-accept-encodings). That will also suss out any that were missed, except for the tests that include an encoding cookie as part of a mock module.